packages/run-protocol/src/econ-behaviors.js

```diff
@@ -251,6 +253,7 @@ export const startVaultFactory = async (
       recordingPeriod: SECONDS_PER_DAY,
     },
   } = {},
+  minInitialDebt = 5_000_000n,
```

Comment:
I was forced into this because some test in `packages/vats` depends on econ-behavior, and doesn't specify any parameters, so the defaults have to work.